libs/vkd3d/resource.c

```diff
@@ -2819,6 +2825,20 @@ static HRESULT d3d12_resource_create(struct d3d12_device *device, uint32_t flags
             FIXME_ONCE("ReadFromSubresource may be buggy on host-visible images with ALLOW_SIMULTANEOUS_ACCESS | ALLOW_UNORDERED_ACCESS.\n");
     }
 
+    if ((flags & VKD3D_RESOURCE_COMMITTED) &&
+        (device->vk_info.EXT_pageable_device_local_memory) &&
```

Comment:
Initial priority only depends on EXT_memory_priority.

Reply:
Also, we tend to check for the feature being enabled, not the EXT since that's the "technically correct" thing to do.

Reply:
Should this consider D3D12_HEAP_FLAG_CREATE_NOT_RESIDENT? NOT_RESIDENT being lowered to 0.0f priority should only happen when pageable_device_local_memory is supported of course.

Reply:
> Should this consider D3D12_HEAP_FLAG_CREATE_NOT_RESIDENT

I believe that's handled elsewhere (at a lower level), I'll check this path actually hits it.

Reply:
> Also, we tend to check for the feature being enabled, not the EXT since that's the "technically correct" thing to do.

Do you mean check for it in VkPhysicalDeviceFeatures2 or something else?  Thanks.